### DISCUSSION TITLE
What major cereal brand has recently changed its name? 🥣🥛

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1215] [Galaxy Trivia] [answered]
**Bonus:** What is a demitasse?

**Comment by peterbaccaro** [comment, marked as answer]
**Kellogg's Australia** recently changed its name to 'Kellanova' as the cereal giant shifts its focus to snacks.

A "demitasse" is **a small coffee cup**.  The French word “demitasse” translates into **half cup**, which is exactly what these diminutive cups are.
